.github/workflows/ci.yml

```diff
@@ -143,7 +143,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python: ['3.7', '3.8', '3.9', '3.10', 'pypy-3.8-nightly', 'pypy-3.9-nightly']
+        python: ['3.8', '3.9', '3.10', 'pypy-3.9-nightly', 'pypy-3.10-nightly']
```

Comment:
Mind adding at least pypy 3.10 here?

Reply:
In addition to `pypy-3.10-nightly`? Ubuntu tests both `pypy-3.10` and `pypy-3.10-nightly`, windows & mac has always only tested pypy-nightlies (no clue why they do that instead of the stable releases though)